src/ext/ddtrace.c

```diff
@@ -55,6 +56,8 @@ STD_PHP_INI_ENTRY("ddtrace.request_init_hook", "", PHP_INI_SYSTEM, OnUpdateStrin
                   zend_ddtrace_globals, ddtrace_globals)
 STD_PHP_INI_ENTRY("ddtrace.ignore_missing_overridables", "1", PHP_INI_SYSTEM, OnUpdateBool, ignore_missing_overridables,
                   zend_ddtrace_globals, ddtrace_globals)
+STD_PHP_INI_ENTRY("ddtrace.debug_mode", "0", PHP_INI_SYSTEM, OnUpdateBool, debug_mode, zend_ddtrace_globals,
```

Comment:
I love this! ❤️My only concern is that this will introduce two ways to get into debug mode and each of them mean something different. So I'm curious if we want to favor this INI setting over the current `DD_TRACE_DEBUG=true` method. Maybe @labbati has some thoughts on this? :)

Reply:
Maybe https://github.com/DataDog/architecture/pull/295 is interesting? I think you can have all of config file, command line args, and environment variables. And they just have a defined priority if they don't match.

Reply:
I like the idea from @willgittoes-dd (command line args are probably not relevant for php use case but the rest is).
To answer @SammyK, even is they share the same name, I am not sure that mean the same thing 😃. Debug mode (as intended as before this PR) is "print useful info as the app is served" without changing how the underlying extension works, while in this case we are changing the way the underlying extension is working and I would keep the two configs separated. What about changing the name to `ddtrace.dump_backtrace`?

As will said, as soon as we implement all the config options, this can be sourced in different ways.

Reply:
@labbati @SammyK how about `ddtrace.log_backtrace` nad DD_LOG_BACKTRACE env/sapi var ?